config/base/crd/bases/catalogd.operatorframework.io_clustercatalogs.yaml

```
@@ -18,8 +18,8 @@ spec:
     - jsonPath: .status.phase
       name: Phase
       type: string
-    - jsonPath: .metadata.creationTimestamp
-      name: Age
+    - jsonPath: .metadata.catalogContentTimestamp
+      name: UnpackedTime
       type: date
```

If I'm understanding Joe right, we want both:
```suggestion
    - jsonPath: .metadata.catalogContentTimestamp
      name: UnpackedTime
      type: date
    - jsonPath: .metadata.creationTimestamp
      name: Age
      type: date
```